spec/acceptance/php_spec.rb

```diff
@@ -45,7 +53,10 @@
               package_prefix => 'php-',
               settings       => {
                 extension => undef
-              }
+              },
+            }
```

Comment:
There is a comma missing:
```suggestion
            },
```